hack/testing/test-upgrade.sh

```diff
@@ -159,7 +159,10 @@ function upgrade() {
                         -p ES_CLUSTER_SIZE=1 \
                         -p PUBLIC_MASTER_URL=https://localhost:8443${masterurlhack} \
                         -p MODE=upgrade \
-                        -p IMAGE_VERSION=$version
+                        -p IMAGE_VERSION=$version \
+                        -p USE_JOURNAL=${USE_JOURNAL:-false} \
```

Comment:
Nit picking here, but do we need to set these as parameters since we're creating the logging-deployer configmap with these settings?